docker_images/diffusers/app/pipelines/text_to_image.py

```diff
@@ -38,9 +44,6 @@ def __init__(self, model_id: str):
         self.ldm = DiffusionPipeline.from_pretrained(
             model_to_load, use_auth_token=os.getenv("HF_API_TOKEN"), **kwargs
         )
-        if torch.cuda.is_available():
```

Comment:
@pcuenca, @patrickvonplaten @radames I would need your advice here: is there any requirement for the previous specif order of operations ? I mean: does to('cuda') need to be called before loading lora weights and assigning the scheduler or it this current proposal ok ?

Reply:
Both work :-)